casual/casual-api/src/test/groovy/se/laz/casual/api/util/work/RepeatUntilSuccessTaskWorkTest.groovy

```diff
@@ -51,7 +51,7 @@ class RepeatUntilSuccessTaskWorkTest extends Specification
          workManager
       }
       when:
-      RepeatUntilSuccessTaskWork taskWork = RepeatUntilSuccessTaskWork.of(supplier, consumer, workManagerSupplier)
+      RepeatUntilSuccessTaskWork taskWork = RepeatUntilSuccessTaskWork.of(supplier, consumer, workManagerSupplier, 1000)
```

Comment:
I think it would be nice to have an additional assertions and/or test cases here to see that the backoff is actually working.
Potentially with different values for the backoff.